webrtc-stats.html

```diff
@@ -1889,6 +1913,15 @@
                   or UDP. It is similar to the TIAS defined in [[!RFC3890]], i.e., it is measured
                   in bits per second and the bitrate is calculated over a 1 second window.
                 </p>
+                <p>
+                  Implementations that do not calculate a sender-side estimate should leave this undefined.
```

Comment:
I'd prefer MUST. There is no circumstance wherein it makes sense to give a value for this member if you don't calcluate it.